debezium-storage/debezium-storage-jdbc/src/main/java/io/debezium/storage/jdbc/JdbcConfig.java

```diff
@@ -35,7 +37,15 @@ public class JdbcConfig extends WorkerConfig {
                 .define(OFFSET_STORAGE_TABLE_NAME.name(),
                         ConfigDef.Type.STRING,
                         ConfigDef.Importance.HIGH,
-                        "Name of the table to store offsets");
+                        "Name of the table to store offsets")
+                .define(OFFSET_STORAGE_TABLE_DDL.name(),
```

Comment:
Should not be needed here.

Reply:
WorkerConfig seems to take ConfigDef and Properties as parameters, not sure what needs to be done here.